### PR TITLE
create places model

### PR DESCRIPTION
**Pull request by mattlub**
#40

slight deviations from the spec tbh

### CODE REVIEW

File: db/models/Place.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+
+// placeholders for now
+const categories = ['restaurant, cafe']
+const accesibilityOptions = ['braille menu', 'wheelchair-friendly']
```
Comment (m4v15) [inline db/models/Place.js]: either braille-menu or wheelchair friendly?

**Review by m4v15 — CHANGES_REQUESTED** [review]

File: db/models/Place.js (hunk 1 above)
Comment (mattlub) [reply db/models/Place.js]: politely disagree (@mantagen?)

**Review by m4v15 — APPROVED** [review]

File: db/models/Place.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+
+// placeholders for now
+const categories = ['restaurant, cafe']
+const accesibilityOptions = ['braille menu']
```
Comment (des-des) [inline db/models/Place.js]: Put both of these in a json file called constants

File: db/models/Place.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+const placeSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: String,
+    description: String,
+    address: String,
+    lat: Number,
```
Comment (des-des) [inline db/models/Place.js]: Use this instead: https://docs.mongodb.com/manual/reference/geojson/

File: db/models/Place.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose')
+
+// placeholders for now
+const categories = ['restaurant, cafe']
+const accesibilityOptions = ['braille menu']
+
+const placeSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: String,
```
Comment (des-des) [inline db/models/Place.js]: isRequired

**Review by des-des — CHANGES_REQUESTED** [review]

File: db/models/Place.js (hunk 3 above)
Comment (mattlub) [reply db/models/Place.js]: @des-des it's not listed in Mongoose schema types:

>Following are all validSchema Types.
String
Number
Date
Buffer
Boolean
Mixed
Objectid
Array

Comment (mattlub) [reply db/models/Place.js]: ?

Comment (des-des) [reply db/models/Place.js]: Yeah, fairly confusing.

https://stackoverflow.com/questions/32199658/create-find-geolocation-in-mongoose

https://github.com/Automattic/mongoose/tree/master/examples/geospatial

Are they helpful?

**Comment by mantagen** [comment]
Who wants to be assigned to this?

**Comment by mattlub** [comment]
I've spent way too long looking at this- I think I've changed it to legacy coordinates, and I think it works ok for now.
https://docs.mongodb.com/manual/geospatial-queries/#legacy-coordinate-pairs

see: https://github.com/Automattic/mongoose/wiki/3.6-Release-Notes#geojson-support-mongodb--24

**Review by des-des — APPROVED** [review]

**Pull request merged**